en-US/Design.xml

```diff
@@ -743,17 +743,27 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
 							In examples or scenarios, you can use a person's name and then use a gender-specific pronoun to refer to that name. Vary the use of proper names in documentation. Use names that represent various ethnic backgrounds, genders, and locations.
 						</para>
 
+					</listitem>
+					 <listitem>
+						<para>
+							Include a diverse set of names in your examples to reflect the diversity of the real world.
+              For example, use male, female, and culturally diverse names that suggest various backgrounds in examples to avoid implying that only certain groups have specific skills.
+						</para>
+
 					</listitem>
 					 <listitem>
 						<para>
 							Do not use copyrighted fictional characters in examples, and do not use real people.
 						</para>
 
 					</listitem>
+
 					 <listitem>
 						<para>
-							Include a diverse set of names in your examples to reflect the diversity of the real world.
-              For example, use male, female, and culturally diverse names that suggest various backgrounds in examples to avoid implying that only certain groups have specific skills.
+							When choosing names, also consider how those names might appear in email addresses, usernames, and similar contexts.
```

Comment:
Based on David's explanation, maybe we could add a line after 763 that says something like: "Avoid name combinations or abbreviations that result in unintended meanings, such as slang." 

This might be kind of obvious, but seems worth explaining directly (along with the examples).